server_api.py

```diff
@@ -609,12 +612,12 @@ def store_thumbnail_image(file_path,thumbnail_path):
 
 def get_img_meta(img_id):
     try:
-        db = mysql()
-        db.connect()
-        sql = 'select * from image_data where img_is_delete = 0 and img_id = "%s"' % img_id
-        return db.query(sql)[0]
+        with ImageDao() as imageDao:
+            img_data = ImageDao.getImgData(imgId=img_id)
+        if img_data != None:
```

Comment:
You can just return `None`, when the `img_data` from getImgData() is None.
And I hope you can also check the return value of get_img_meda() in caller function to make sure it won't cause another problem when the img_data is not existed.